cvat/apps/engine/notify_jobflow.py

```diff
@@ -23,5 +23,5 @@ def jobflow_notifier(db_jobs, db_task):
         "task_address": db_task.name,
         "payouts": payouts
     }
-    requests.post(NOTIFY_JOBFLOW_URL, json=payload)
+    requests.post(f'{JOBFLOW_URL}/notify', json=payload)
```

Comment:
please, check response status code and response message
in case of job flow error your application won't say anything